libraries/tuttle/src/tuttle/host/graph/GraphExporter.tcc

```diff
@@ -5,23 +5,27 @@
 #include <boost/graph/graphviz.hpp>
 #include <iostream>
 
-namespace tuttle {
-namespace host {
-namespace graph {
+namespace tuttle
+{
```

Comment:
We have this formatting because of the following option:
__BreakBeforeBraces: Allman__